TLM/TLM/Patcher.cs

```diff
@@ -0,0 +1,93 @@
+namespace TrafficManager {
+    using ColossalFramework;
+    using ColossalFramework.UI;
+    using CSUtil.Commons;
+    using HarmonyLib;
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using TrafficManager.RedirectionFramework;
+    using TrafficManager.Util;
+
+    public class Patcher {
+        public static Patcher Instance { get; private set; }
+
+        public static Patcher Create() => Instance = new Patcher();
+
+        private const string HARMONY_ID = "de.viathinksoft.tmpe";
```

Comment:
Should we consider changing this to `me.tmpe`?

Reply:
NO!!!!!!!!!!!!!
I am using this in my mods. You should never change such stuff.
[HarmonyBefore] and [HarmonyAfter] require compile time constants so you cannot expect people to write code to evaluate such IDs runtime.

Reply:
Of course we should finally change that name. 
First, you've created that class, it was not used before - there is no mod that would use it before. (It's sort of id used by harmony internally to figure out which mod applied which patch/prefix/transpiler (who was the author))
Second, this field is `private` which means: hidden, value not meant to be used by anyone else outside that class. No one should even care what value is there. It's private!

Reply:
@krzychu124 please see #897 to discuss what to do with harmony ID. I don't want it to be part of this review please.

To answer your questions.

> First, you've created that class, it was not used before

I moved Harmony ID string from LoadingExtension.cs I didn't add any code I just moved/deleted code.

> It's sort of id used by harmony internally

No its more than that. see #897

> there is no mod that would use it 

My node controller that uses it.  see #897

> Second, this field is private which means: hidden, value not meant to be used by anyone else outside that class. No one should even care what value is there. It's private!

Maybe I should make it public or move it to TMPE.API see #897